packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -260,6 +257,108 @@ export default async function main(
     );
   }
 
+  const client = new Client({
+    apiUrl: ctx.apiUrl,
+    token: ctx.authConfig.token,
+    debug: debugEnabled,
+  });
+
+  // retrieve `project` and `org` from .now
+  const link = await getLinkedProject(output, client, path);
+
+  if (link.status === 'error') {
+    return link.exitCode;
+  }
+
+  let { org, project, status } = link;
+  let newProjectName = null;
+  let rootDirectory = project ? project.rootDirectory : null;
+
+  if (status === 'not_linked') {
+    const shouldStartSetup =
+      autoConfirm ||
+      (await confirm(
+        `Set up and deploy ${chalk.cyan(`“${toHumanPath(path)}”`)}?`,
+        true
+      ));
+
+    if (!shouldStartSetup) {
+      output.print(`Aborted. Project not set up.\n`);
+      return 0;
+    }
+
+    org = await selectOrg(
+      output,
+      'Which scope do you want to deploy to?',
+      client,
+      ctx.config.currentTeam,
+      autoConfirm
+    );
+
+    // We use `localConfig` here to read the name
+    // even though the `now.json` file can change
+    // afterwards, this is fine since the property
```

Comment:
What do you mean by "change afterwards"? Where does it change and how?

Reply:
Are you referring to line 359? 

```
localConfig = readLocalConfig(sourcePath) || localConfig;
```

Reply:
@styfle yes